src/Capture/OSX.hs

```diff
@@ -16,9 +16,12 @@ captureData = do
         titles <- fetchWindowTitles
         foreground <- getForegroundWindow
 
-        let winData = map (
-                \(h,t,p) -> (h == foreground, T.pack t, T.pack p)
-                ) titles
+        let winData = do
+                (h, t, p) <- titles
+                return def{ wActive = h == foreground
+                          , wTitle = T.pack t
+                          , wProgram = T.pack p
+                          }
```

Comment:
This is lists? Then I usually find list comprehensions nicer to read (and makes it easier to see that these indeed are just lists and not some fancy monad)

Reply:
Yeah, I was trying to indent this sensibly and totally forgot about comprehensions. Dunno what was going on in my head back then. I'll adjust this. :-)